updating/preparing_for_updates/updating-cluster-prepare.adoc

```diff
@@ -12,31 +12,42 @@ WARNING: This assembly has been moved into a subdirectory for 4.14+. Changes to
 To do: Remove this comment once 4.13 docs are EOL.
 ////
 
+Learn more about administrative tasks that cluster admins must perform to successfully initialize an update, as well as optional guidelines for ensuring a successful update.
+
+== Kubernetes API deprecations and removals
```

Comment:
@skopacz1, I believe this needs an `id` even as a level 2, such as:

```
[id="{context}_kubernetes-api-deprecations-and-removals"]
```

Reply:
Good call, thanks!